waku/waku_keystore/conversion_utils.nim

```diff
@@ -18,6 +18,16 @@ proc decode*(encodedCredential: seq[byte]): KeystoreResult[KeystoreMembership] =
     let jsonObject = parseJson(string.fromBytes(encodedCredential))
     return ok(to(jsonObject, KeystoreMembership))
   except JsonParsingError:
-    return err(AppKeystoreError(kind: KeystoreJsonError, msg: getCurrentExceptionMsg()))
+    return err(
+      AppKeystoreError(
+        kind: KeystoreJsonError,
+        msg: "error in conversion_utils decode: " & getCurrentExceptionMsg(),
```

Comment:
If it will be for the public ... like logs for our node-runners I would rather formalize like:
`error during decoding credentials:`